lib/trento_web/controllers/v1/host_controller.ex

```diff
@@ -191,15 +192,12 @@ defmodule TrentoWeb.V1.HostController do
     %{solution: solution} = OpenApiSpex.body_params(conn)
     %{id: host_id} = host
 
-    Logger.info(
-      "Operation authorized. Send real saptune_solution_apply with #{solution} to #{host_id}"
-    )
-
-    # Request operation, get operation id when created and return in json
-
-    conn
-    |> put_status(:accepted)
-    |> json(%{operation_id: UUID.uuid4()})
+    with {:ok, operation_id} <-
```

Comment:
I think we have this pattern with a single with in controllers, but if you want to make more explicit, the case statement also here is a good idea

Reply:
I will keep this using the `with`. I think it is the standard way of using the fallback controller and we have it all around the controllers